linux/ZramMeter.c

```diff
@@ -0,0 +1,54 @@
+#include "ZramMeter.h"
+
+#include "Meter.h"
+#include "CRT.h"
+#include "Platform.h"
+
+int ZramMeter_attributes[] = {
+   ZRAM
+};
+
+static void ZramMeter_updateValues(Meter* this, char* buffer, int size) {
+   int written;
+
+   Platform_setZramValues(this);
+   written = Meter_humanUnit(buffer, this->values[0], size);
+
+   buffer += written;
+   *buffer++ = '(';
```

Comment:
`size` needs to be decreased by `written` and checked `> 0`, cause the buffer might end.

side note: In the long run we might want to refactor `Meter_humanUnit` to clean up such code; affects also `MemoryMeter_updateValues()` and `SwapMeter_updateValues()`